caluma/caluma_workflow/events.py

```diff
@@ -0,0 +1,12 @@
+from django.dispatch import Signal
+
+new_work_item = Signal(providing_args=["work_item", "info"])
```

Comment:
It might be useful to name the events using a past-tense verb to imply more strongly that the events are triggered after the completion of the change. For example `saved_work_item`, `created_work_item` and so on.

BTW What's the difference between `new_work_item` and `create_work_item`? Seeing from the code, one seems to be when created implicitly via starting a new case, the other when creating one explicitly. Could this be encoded into a parameter instead?

Reply:
> BTW What's the difference between new_work_item and create_work_item? Seeing from the code, one seems to be when created implicitly via starting a new case, the other when creating one explicitly. Could this be encoded into a parameter instead?

I would go one step further: Let's have just one `created_work_item` signal. The sender makes it clear why the event was emitted.

Reply:
Indeed, that's even better!

Reply:
Done.